Automattic/jetpack
# Private Site: remove from Jetpack 7.4

#### Changes proposed in this Pull Request:

* As the feature is not ready for prime-time yet, let's consider removing it from Jetpack 7.4 for now.

Internal reference: p1559248287075600-slack-jetpack-crew

#### Testing instructions:

* Checkout this branch
* Do you see any tracks of the Private features in Settings > Reading, the admin dashboard, Jetpack > Settings?
* Can you activate and deactivate other modules?

#### Proposed changelog entry for your changes:

* None
